src/modules/I18n/i18n.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   pages: {
     '*': ['common', 'header', 'footer', 'follow-us'],
     '/': ['homepage', 'instances'],
+    'rgx:/case-studies': ['case-studies'],
```

Comment:
I don't understand what is `rgx:`. Maybe that could benefit from a comment or a clearer name? 🙂

Reply:
This is part of the next-translate [configuration](https://github.com/aralroca/next-translate#3-configuration), I document.